lib/main.dart

```diff
@@ -94,6 +94,11 @@ ${parser.usage}
     designType = 'figma';
   }
 
+  if (projectName == 'test') {
```

Comment:
Hey @snwagbata I really like the idea of checking for invalid names before running parabeac-core. However, I do not think we should do this in `main.dart`, since in the future we could add more invalid names. I suggest adding maybe a separate class to detect for valid/invalid names would be a lot better, what do you think? 🤔

Reply:
That sounds like a great idea! Where would be the ideal location for having the class?

Reply:
Currently verifying this with the team! I'll get back to you with an answer soon!

Reply:
Hi @snwagbata, apologies for the delay. I think the best place for this file would be in the `lib/controllers` directory. Perhaps it could be named something like `project_name_verifier.dart` or something along those lines. Let me know what you think or if you have any other ideas!

Reply:
Hey @ivan-015, I went with `project_name_validator.dart` for clarity. Also, I discovered Dart's linter package could handle package name validation, so I went with that. Nevertheless, I also implemented a way for adding invalid names in the future.